### PR TITLE
add gofmt call to go:generate directive

### PR DESCRIPTION
**Pull request by galargh**
`gofmt -s -l .` is a part of the CI workflow. It currently fails on `gen/tmplgen/bindata.go` file which blocks the CI sync from getting merged: https://github.com/ipld/go-ipld-graphql/pull/34

In this PR I've added a command to run `gofmt` on generated file to `go:generate` directive and run `gofmt` myself to fix the issue with the currently generated/commited file.

### CODE REVIEW

**Comment by galargh** [comment]
@mvdan WDYT?

**Review by willscott — APPROVED** [review]

**Comment by willscott** [comment]
sorry missed this earlier. this looks reasonable to me 👍

**Comment by galargh** [comment]
> sorry missed this earlier. this looks reasonable to me 👍

Thank you 🙇  FYI, I don't have write access to this repository so I cannot merge this myself :(

**Comment by mvdan** [comment]
> @mvdan WDYT?

Just getting to this now :) How about just using `go:embed`, added in Go 1.16? It's simpler, faster, and removes the need for third-party tools like go-bindata. I don't think requiring Go 1.16 or later is a problem at this point, given that it came out a year ago, and we generally have a policy of supporting two Go versions.

**Pull request merged**
